cypress/support/commands.js

```diff
@@ -22,4 +26,45 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+const workerDate = {
```

Comment:
The object name 'workerDate' seems to be a typo. It should probably be 'workerData'.